README.md

```diff
@@ -1,25 +1,46 @@
 [![Documentation Status](https://readthedocs.org/projects/deepcell-kiosk-figure-generation/badge/?version=latest)](https://deepcell-kiosk-figure-generation.readthedocs.io/en/latest/?badge=latest)
 
-Recreate the figures from https://www.biorxiv.org/content/10.1101/505032v3 in 4 easy steps!
+The DeepCell Kiosk paper [(Bannon et al., 2020)](<https://www.biorxiv.org/content/10.1101/505032v3>) presents cost and runtime benchmarks for running a generic image segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b). This repository exist to aid DeepCell Kiosk users in recreating these figures.
+ 
+## Figure Creation
+
+There are two steps to figure creation:
+    1) Generating a full battery of benchmark data for the DeepCell Kiosk under a variety of conditions. See the `Generating Benchmarking Data` section below for details
+    2) Creating the figures from this benchmarking data. See the `Figure Creation` section below for details
```

Comment:
Formatting on list seems off. Probably don't need the headline as the repo is called Publication Figures. And given the length of the README, I don't think you need: "see ... below for details."

Reply:
Both good points.